lib/main.ts

```diff
@@ -8,4 +8,4 @@ export { RadioButton } from './components/RadioButton/RadioButton';
 export { RadioButtonGroup } from './components/RadioButton/RadioButtonGroup';
 export { RoundButton } from './components/RoundButton/RoundButton';
 export { RoundLinkButton } from './components/RoundLinkButton/RoundLinkButton';
-export { TextCard } from './components/TextCard/TextCard';
+export { ResultsCard } from './components/ResultsCard/ResultsCard';
```

Comment:
Should we try to keep the component listed here in alphabetical order?

Reply:
Fixed